docker/Dockerfile

```diff
@@ -1,4 +1,11 @@
-FROM ubuntu:15.04
+# This base image is equivalent to Ubuntu 16.04 but has better defaults
+# and fixes numerous problems with the stock Ubuntu images.
+# read more about it here: https://github.com/phusion/baseimage-docker
+FROM phusion/baseimage:0.9.19
```

Comment:
Regarding the failure of 15.04. I tried the build a week ago, and it failed. I also tried looking for the image or docker hub and could not find it. I think you could build on it because you have a local copy of the image. That's the main reason considered 16.04. Just rechecked, 15.04 is not in the supported tag anymore: https://hub.docker.com/_/ubuntu/

I tested with the official 16.04 on `viral-ngs` repo, and faced many problems which I don't remembers all, but the most notable one was that ansible not able to run because its python2 dependency was not installed in the base image. But I've  just tried with this deploy repo and it seems working fine. This mainly because it drops `ansible`. So I actually don't have a compelling reason to persuade the switch. However, still have trust on the `phusion/baseimage` because many github users are using it, and contributing to it (1M+ image pulls, 5k+ github stars, 500+ github forks). So it is a more proactive to deal with problems, and allows users to have more advanced uses with the built image. In short, it is a safe bet. With that said, I agree that there is not a compelling reason to switch.